### PR TITLE
Added animated splash screen

### PR DESCRIPTION
**Pull request by prasad-more-12**
## What type of PR is this? (check all applicable)

- [ ] 🚀 Added Name
- [ ] ✨ Feature
- [ ] 🌟 stared the repo
- [ ] 🐛 Grammatical Error
- [ ] 📝 Documentation Update
- [ ] 🚩 Other

### CODE REVIEW

**Review by AndroidLion48 — COMMENTED** [review]
May you upload a short GIF recording of the splash screen? 
Thanks in advance, and we appreciate your awesome contributions 😃! 
@prasad-more-12

**Review by AndroidLion48 — CHANGES_REQUESTED** [review]
Add GIF Recording

**Comment by prasad-more-12** [comment]
@AndroidLion48 ok i will actually it was night time here.

**Comment by prasad-more-12** [comment]
@AndroidLion48 now check.

**Comment by prasad-more-12** [comment]
@AndroidLion48 all done now check.

**Comment by Tc2r1** [comment]
When leaving a comment here, if you look Just below the comment window you can click "attach files by dragging" and it lets you import an image or video @prasad-more-12

**Comment by prasad-more-12** [comment]
Anim Splash Screen

**Comment by prasad-more-12** [comment]
ok thanks i didn't know I was supposed to add GIF here not in repo. Should I revert my last commit ?

**Comment by AndroidLion48** [comment]
Yes, please revert your commit of the add splash screen file. We only wanted to review what the changes looked like. I love the splash screen animation by the way.  Really great work 😃! 

My apologies for not being clear enough on my request for changes.  Thank you for your time and efforts 😄 👍

File: Interview_Prep_App/native_java/retrofit/src/main/res/values/strings.xml [hunk 1]
```diff
@@ -0,0 +1 @@
+<resources></resources>
```
Comment (Tc2r1) [inline Interview_Prep_App/native_java/retrofit/src/main/res/values/strings.xml:1]: Please remove this file.

**Comment by prasad-more-12** [comment]
Sorry for messy PR my branch became scuffed so i have to force push from my local changes. @Tc2r1 It should be good know and  thanks.

File: Interview_Prep_App/native_java/retrofit/src/main/res/layout/activity_anim_splash_screen.xml [hunk 2]
```diff
@@ -4,31 +4,31 @@
     xmlns:tools="http://schemas.android.com/tools"
```
Comment (Tc2r1) [inline Interview_Prep_App/native_java/retrofit/src/main/res/layout/activity_anim_splash_screen.xml:4]: Please restore the original file here and create a new xml to link yours to. I'd like to save other's work in the project

File: Interview_Prep_App/native_java/retrofit/src/main/java/com/dreams/interviewprepapp/ui/main/SplashScreenActivity.java [hunk 3]
```diff
@@ -1,28 +0,0 @@
-package com.dreams.interviewprepapp.ui.main;
```
Comment (Tc2r1) [inline Interview_Prep_App/native_java/retrofit/src/main/java/com/dreams/interviewprepapp/ui/main/SplashScreenActivity.java:1]: Ditto, please restore this file and simply change the manifest to use yours as launcher instead.

**Review by Tc2r1 — APPROVED** [review]

File: Interview_Prep_App/native_java/retrofit/src/main/java/com/dreams/interviewprepapp/ui/main/SplashScreenActivity.java (hunk 3 above)
Comment (prasad-more-12) [reply Interview_Prep_App/native_java/retrofit/src/main/java/com/dreams/interviewprepapp/ui/main/SplashScreenActivity.java:1]: Can you tell me what should i do. Should i create these files myself or should reset my commit before i deleted them.

Comment (Tc2r1) [reply Interview_Prep_App/native_java/retrofit/src/main/java/com/dreams/interviewprepapp/ui/main/SplashScreenActivity.java]: @prasad-more-12  Hi, I think you should reset your commit to before you deleted them, do a soft reset imo would be best.

**Review by Tc2r1 — APPROVED** [review]

**Pull request merged**